cmake/Modules/FindCUDAToolkit.cmake

```diff
@@ -978,6 +978,14 @@ if(CUDAToolkit_FOUND)
     _CUDAToolkit_find_and_add_import_lib(cublas_static DEPS culibos)
   endif()
 
+  if(CUDAToolkit_VERSION VERSION_GREATER_EQUAL 11.4)
```

Comment:
Is this file vendored from somewhere? Did you take that update from there?

Reply:
yep from here https://gitlab.kitware.com/cmake/cmake/-/blob/master/Modules/FindCUDAToolkit.cmake#L1238-1244

Reply:
Ok!
Did you update the full thing to 3.25? Or just picked the subset of the changes you needed?
@malfet what do you think is the best way to do this? A full update to a given version to make sure we're not in a weird in-between. Or just what we need to reduce churn?

Reply:
Just copy pasted, fwiw I am not sure that the initial file was copy pasted directly as I see this at the top https://github.com/pytorch/pytorch/blob/e12cfe3546af7609bce77f9cc196c7381a3e7503/cmake/Modules/FindCUDAToolkit.cmake#L2

not sure whether there were any changes made/which commit to CMake it was taken from